help/en/html/web/JsonServlet.shtml

```diff
@@ -1,17 +1,17 @@
 <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
 <html lang="en">
     <head>
-        <!-- Copyright 2012 -->
+        <!-- Copyright 2015 -->
         <!-- $Id$ -->
         <title>JMRI JSON Protocol</title>
 
         <!-- Style -->
         <meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
-        <link rel="stylesheet" type="text/css" href="/css/default.css" media="screen">
+        <link rel="stylesheet" type="text/css" href="/css/default.css" mediaedia="screen">
```

Comment:
I’m sorry,  should have checked that one better.
I’ve just added 1 commit to the Branch; does that come through?

Egbert

> Op 14 okt. 2015, om 23:11 heeft Matthew Harris <notifications@github.com> het volgende geschreven:
> 
> In help/en/html/web/JsonServlet.shtml <https://github.com/JMRI/JMRI/pull/92#discussion_r42053808>:
> 
> >          <!-- $Id$ -->
> >          <title>JMRI JSON Protocol</title>
> >  
> >          <!-- Style -->
> >          <meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
> > -        <link rel="stylesheet" type="text/css" href="/css/default.css" media="screen">
> > +        <link rel="stylesheet" type="text/css" href="/css/default.css" mediaedia="screen">
> @silverailscolo <https://github.com/silverailscolo> 
> This doesn't look right
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/JMRI/JMRI/pull/92/files#r42053808>.
>